libcst/_nodes/statement.py

```diff
@@ -2881,6 +2889,15 @@ def lpar(self, value: Sequence[LeftParen]) -> None:
         # pyre-fixme[41]: Cannot reassign final attribute `lpar`.
         self.value.lpar = value
 
+    @property
+    def rpar(self) -> Sequence[RightParen]:
+        return self.value.rpar
+
+    @rpar.setter
+    def rpar(self, value: Sequence[RightParen]) -> None:
+        # pyre-fixme[41]: Cannot reassign final attribute `rpar`.
+        self.value.rpar = value
```

Comment:
hmm, in retrospect I think we don't need these setters, these dataclasses are frozen. I'll follow up.